jupyterhub/notebook-images/overlays/odh-ray-cluster/operator/ray-operator-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ray-operator
+spec:
+  # Do not change: currently only 1 replica is supported
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ray-operator
+  template:
+    metadata:
+      labels:
+        app: ray-operator
+    spec:
+      restartPolicy: Always
+      serviceAccountName: ray-operator
+      containers:
+        - name: ray-operator
+          imagePullPolicy: Always
+          image: 'quay.io/thoth-station/ray-operator:v0.1.2'
```

Comment:
Should the image be in the Open Data Hub organization?

Reply:
(even if it's just a mirror of the image in Thoth-station)

Reply:
It just that, if we keep open-data-org image in here, people facing issue would have to look for the image in here.
i m happy to make the change, if that is way we want to go.